packages/core/src/features/table/table-component.injectable.ts

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
Move this under `open-lens` package and import `Table` from `@k8slens/core`.  Add export if it's not exported yet.

Reply:
@aleksfront